src/alloc_pool.c

```diff
@@ -90,7 +90,7 @@ DEQ_DECLARE(qd_alloc_type_t, qd_alloc_type_list_t);
 #if QD_MEMORY_STATS
 static const char *leaking_types[] = {
     // DISPATCH-1679:
-    "qd_timer_t", "qd_connector_t",
+    //"qd_timer_t", "qd_connector_t",
```

Comment:
Just delete the two lines. We could add a link in the JIRA DISPATCH-2046 about DISPATCH-1679